sdk/servicebus/perf-tests/service-bus/test/receiveBatch.spec.ts

```diff
@@ -31,8 +30,7 @@ export class BatchReceiveTest extends ServiceBusTest<ReceiverOptions> {
     "max-message-count": {
       required: true,
       description: "Max number of messages to receive",
-      shortName: "max",
-      longName: "maximum",
+      shortName: "max-receive",
       defaultValue: 50,
```

Comment:
So, `optionName`, `longName` and `shortName`.

If `longName` and `shortName` are present, `optionName` can't be used in the command.
If only `shortName` is provided, `optionName` is treated as the `longName`.

Is this right?

Reply:
Yes, `longName` defaults to the name of the option when it's not specified.